Extensions/RosettaPython/Sources/Python/Cards/Cards.cpp

```diff
@@ -0,0 +1,280 @@
+// Copyright (c) 2019 Chris Ohk, Youngjoong Kim, SeungHyun Jeon
+
+// We are making my contributions/submissions to this project solely in our
+// personal capacity and are not conveying any rights to any intellectual
+// property of any third parties.
+
+#include <Python/Cards/Cards.hpp>
+#include <Rosetta/Cards/Cards.hpp>
+
+#include <pybind11/pybind11.h>
+#include <pybind11/stl.h>
+
+using namespace RosettaStone;
+
+using CardListCasterBase =
+    pybind11::detail::list_caster<std::vector<Card*>, Card*>;
+
+namespace pybind11::detail
+{
+template <>
+struct type_caster<std::vector<Card*>> : CardListCasterBase
+{
+    static handle cast(const std::vector<Card*>& src, return_value_policy,
+                       handle parent)
+    {
+        return CardListCasterBase::cast(src, return_value_policy::reference,
+                                        parent);
+    }
+
+    static handle cast(const std::vector<Card*>* src,
+                       return_value_policy policy, handle parent)
+    {
+        return cast(*src, policy, parent);
+    }
+};
+}  // namespace pybind11::detail
```

Comment:
What is this? 💭

Reply:
To resolve "Invalid map<K, V> key" error, I wrote above code.
Without this code, python may be destroy Card instance.